configs/h2rbox_v2/README.md

```diff
@@ -0,0 +1,50 @@
+# H2RBox-v2
```

Comment:
Update the Model Zoo in readme https://github.com/open-mmlab/mmrotate/tree/1.x#model-zoo